3p/beopinion.js

```diff
@@ -87,6 +87,9 @@ function createContainer(global, data) {
  */
 function getBeOpinionAsyncInit(global, accountId) {
   const {context} = global;
+  context.onResizeDenied(function(requestedHeight, requestedWidth) {
+    context.requestResize(requestedWidth, requestedHeight);
```

Comment:
woops. this is pretty brutal. could you add some timeout?

Reply:
never mind, we should add the protection in API: https://github.com/ampproject/amphtml/issues/25894